src/Components/Scorecard.js

```diff
@@ -1,150 +1,114 @@
 import React, { useContext, useState, useEffect } from 'react'
 import { ScorecardInfo } from '../Contexts/ScorecardInfoProvider'
+import { CourseInfo } from '../Contexts/CourseInfoProvider'
 import { Errors } from '../Contexts/ErrorsProvider'
 import Player from './Player'
 import { scoreAHole, endRound } from '../apiCalls'
 import {useNavigate} from 'react-router-dom'
 
 const Scorecard = () => {
   const { scorecard, setScorecard } = useContext(ScorecardInfo)
-  const [ currentHole, setCurrentHole ] = useState({})
+  const [ currentHole, setCurrentHole ] = useState({par: 3, number: 1, distance: 0})
   const {errorMessage, setErrorMessage} = useContext(Errors)
   const [inProgress, setInProgress] = useState(true)
+  const [scores, setScores] = useState([])
+  const {courseInfo, setCourseInfo} = useContext(CourseInfo)
```

Comment:
So glad we kept the course info in the courseInfo context instead of shoving everything in the scorecard context